k8s/plugin/README.md

```diff
@@ -334,10 +334,31 @@ SUBCOMMANDS:
    ```
 
 3. To upgrade the installation
-   ```sh
-   ## Command
-   kubectl mayastor upgrade
-   ```
+```sh
+  ## Command
+  kubectl mayastor upgrade
+  `Upgrade` the deployment
+
+  Usage: kubectl-mayastor upgrade [OPTIONS]
+
+  Options:
+    -r, --rest <REST>
+        The rest endpoint to connect to
+    -u, --upgrade-operator-endpoint <UPGRADE_OPERATOR_ENDPOINT>
+        Endpoint of upgrade operator service, if left empty then it will try to parse endpoints from upgrade operator service(K8s service resource)
+    -d, --dry-run
+        If set then it will display all the validations o/p but wont execute upgrade
```

Comment:
`display all the validations o/p but wont execute upgrade` , we don't need the  `If set then it will`